packages/cli/src/lib/platform/authenticatedClient.ts

```diff
@@ -65,6 +65,14 @@ export class AuthenticatedClient {
       ],
     });
   }
+
+  async getUserInfo() {
+    const authenticatedClient = new AuthenticatedClient();
+    const platformClient = await authenticatedClient.getClient();
+    await platformClient.initialize();
```

Comment:
Why do we need to initialize here ?

Reply:
Because, if I don't pass any argument to the `user.get()` method, it tries to load the currentUser (which does not exist at that moment)
https://github.com/coveo/platform-client/blob/master/src/resources/Users/User.ts#L8